.github/workflows/integration_test.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         tox -e test-build"
       modules: '["test_core", "test_error", "test_upgrade"]'
       extra-arguments: |
-        -m "not (requires_secret)" \
+        -m "not (require_secret)" \
```

Comment:
`requires_secret` seems to make more sense, in my original comment it was on the other requires in the line, I'll clarify that comment

Reply:
Oh I see! Thank you for the clarification! Just landed in Helsinki :D Will be fixed promptly with other type hints.